docs/usage/verticles/EntityVerticle.md

```diff
@@ -0,0 +1,35 @@
+# EntityVerticle
+
+# Create
+
+A `POST` request will be used to create a new entity. In this case, the `Future<EntityWrapper> createData(DataQuery query, DataContext context)` method of the responsible `EntityVerticle` will be invoked.
+According to the best-practices of REST-API, the response of such a POST-request should
+* contain a `Location`-header with the location of the newly created entity
+* return the entity representation in the response body
+
+To achieve this, a `EntityVertile` should in the `createEntity`-method
+* set the `Location` in the response context data
+```
+context.responseData().put("Location", /io.neonbee.test3.TestService3/TestCars(/unique-id));
```

Comment:
When we send back a filled EntityWrapper, we have the entity type informatiomn and the unique-id available in the EntityProcessor. Why not simply "generate" the location handler? No need to send also the location header in response data.

Reply:
It depends how the URL convention is in the application, where the EntityVerticle is deployed.

For FORME, a URL like /backend/odata/namespace/type(key) should be returned, so that frontend can really use this header. 

Other application might choose to use other prefixes or no prefix at all.

Reply:
I'm not convinced. The URL Convention is configured in the ServerVerticle / Endpoint, on the web-facing node. So the web-facing node should be ideal to "generate" the correct URL.

In my opinion the EntityVerticle is the wrong place to "generate" the location. Let us discuss this in the next committer meeting.